components/Dashboard/Form/Field.tsx

```diff
@@ -31,13 +32,19 @@ export default function Field<T extends FieldValues = FieldValues>({
   control,
   name,
   valueProps,
+  labelProps,
   ...rest
 }: Props<T>) {
   return (
     <div className="flex flex-col gap-y-1">
       <div className="flex flex-col">
         {label && (
-          <label className={heading({ size: 'xs' })}>
+          <label
+            className={heading({
+              size: 'xs',
+              className: labelProps?.className,
+            })}
+          >
```

Comment:
Maybe we don't need to use `heading` style because it uses another font than `Inter`, instead just use regular `className` prop and prioritize `labelProps.className` if any

```suggestion
            className={labelProps?.className ?? "default label class"}
          >
```

Reply:
nice, already did it